src/wallet/asyncrpcoperation_sendmany.cpp

```diff
@@ -270,7 +275,7 @@ bool AsyncRPCOperation_sendmany::main_impl() {
     }
 
     LogPrint("zrpc", "%s: spending %s to send %s with fee %s\n",
-            getId().substr(0,10), FormatMoney(targetAmount, false), FormatMoney(sendAmount, false), FormatMoney(minersFee, false));
+            getId(), FormatMoney(targetAmount, false), FormatMoney(sendAmount, false), FormatMoney(minersFee, false));
     LogPrint("zrpc", " -  transparent input: %s (to choose from)\n", FormatMoney(t_inputs_total, false));
     LogPrint("zrpc", " -      private input: %s (to choose from)\n", FormatMoney(z_inputs_total, false));
     LogPrint("zrpc", " - transparent output: %s\n", FormatMoney(t_outputs_total, false));
```

Comment:
Here's an example of info that should probably be in "zrpcunsafe", but that's outside the scope of this PR.